_1_basics/src/main/java/code/_4_student_effort/FizzBuzz.java

```diff
@@ -0,0 +1,22 @@
+public class FizzBuzz {
+    public static void main(String[] args) {
+        for (int i = 1; i <= 100; i++) {
+            if (i % 3 == 0 || i % 5 == 0 || i % 7 == 0 || i % 11 == 0) {
```

Comment:
crezi ca ai putea gasi o varianta a.i. sa faci verificarea 1 singura data ?
in momentul de fata verifici de 2 ori daca un nr este divizibil cu ceva ...